sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -139,8 +138,7 @@ public CryptographyAsyncClient getCryptographyAsyncClient(String keyName, String
      *
      * <p>The {@link KeyType keyType} indicates the type of {@link KeyVaultKey key} to create. Possible values include:
      * {@link KeyType#EC EC}, {@link KeyType#EC_HSM EC-HSM}, {@link KeyType#RSA RSA}, {@link KeyType#RSA_HSM RSA-HSM},
-     * {@link KeyType#OCT OCT}, {@link KeyType#OCT_HSM OCT-HSM}, {@link KeyType#OKP OKP} and
-     * {@link KeyType#OKP_HSM OKP-HSM}.</p>
+     * {@link KeyType#OCT OCT}, and {@link KeyType#OCT_HSM OCT-HSM}.</p>
```

Comment:
Here and everywhere else: do not use commas to separate only 2 articles.

Reply:
Just using an Oxford comma because in those comments we actually list like 8 things. I personally don't care for using it but some people are really anal about it :P

Reply:
When more than 2 articles, yes, use an Oxford comma; though, this wasn't consistent. Here I misread, I admit, but there were a couple places where now the list only seems to be 2 articles so no comma should be used.

Reply:
Gotcha. We're looking to overhaul our Java documentation across all libraries so I'll take care of those once we get to work on that :)